tools/mkmapdata/mkmapdata.sh

```diff
@@ -446,17 +429,9 @@ if [ "$interactive" = "NO" ]; then
    mpirun=$MPIEXEC
    echo "Running in batch mode"
 else
-   mpirun=""
+   mpirun="mpirun -np 1"
```

Comment:
It would be better for the serial case to NOT be required to run with MPI. Is always running with MPI a requirement of OCGIS?

Reply:
It is not a requirement for ocgis. ESMF must be built with MPI support for IO operations...

Reply:
Thanks for that @bekozi. On cheyenne there are versions of ESMF that are built without MPI (the "uni" versions). So we used them for the serial cases. Can ocgis choose to point to a version of ESMF that is built without MPI?

Reply:
I think we will encounter a problem in the weight file write routine since it uses PIO v1 which ESMF does not wrap with mpiuni. This could be another reason to fall back on the other weight file write routine which, while not writing concurrently, uses netcdf directly so supports mpiuni (I'm 99% sure this is the case). I'll confirm this with the ESMF group.

Reply:
OK, thanks for pointing that out. If we need to require MPI, I think that's OK. But, because it's so much simpler to run without MPI, I like to have it as an option. It's often easier to setup the non-MPI version on a machine first, rather than having to also bring in the extra complexity for MPI. Practically, I'm not sure you can run the regridding for a non-MPI case at least for any sizable grid. It would be hopeful (and helpful) that you could run a single point grid without MPI though for example though.

Reply:
I'll talk with @rokuingh about adding another option for writing weight files in ESMPy. This should allow us to handle the `mpiuni` case. We will also be able to maintain the center coordinates in the weight file for the interim. An option to switch between the write routines will be available. I created an issue on the ocgis side (https://github.com/NCPP/ocgis/issues/511) to track this capability.

I did want to amend my comment on ESMF weight file center coordinates from earlier. The center coordinates in the ESMF weight file are _not_ repeated for each source/destination index combination. The center coordinates are copied over from the source/destination grids. This can be inefficient for larger grids, but not as inefficient as my earlier comment would make it appear.

Ping @oehmke

Reply:
@rokuingh is planning to work on these two features (verbose weight file and arbitrary sequence indices for factor lists) hopefully next week.